src/_designs/screens/index.vue

```diff
@@ -3,33 +3,34 @@
     id="blockchain"
     class="app"
   >
-    <AppNav>
-      <AppNavLink to="dashboard">
-        Dashboard
-      </AppNavLink>
-      <AppNavLink to="generations">
-        Generations
-      </AppNavLink>
-      <AppNavLink to="transactions">
-        Transactions
-      </AppNavLink>
-    </AppNav>
-    <RouterView />
+    <AppHeader />
+    <AppView>
+      <RouterView />
+    </AppView>
   </div>
 </template>
 <script>
-import AppNav from '@/_designs/components/appNav'
-import AppNavLink from '@/_designs/components/appNavLink'
+
+import AppHeader from '@/_designs/partials/appHeader'
+import AppView from '@/_designs/partials/appView'
 
 export default {
   name: 'App',
   components: {
-    AppNav,
-    AppNavLink
+    AppHeader,
+    AppView
   }
 }
 </script>
 <style lang="scss">
 @import '../styles/index';
-.app {}
+.app {
+  background-color: #F7FAFC;
+  padding-bottom: 2rem;
+}
+  h1 {
```

Comment:
these styles will apply globally, are you sure is intended behavior?

Reply:
This is from a PR that was never merged, as it wasn't ready yet. Only intended as a first impression of the global layout. The background color is repeated throughout the app, the padding-bottom and h1 could possibly go.

Reply:
@artem-rozhkov-dev-optima branched out of this instead of the `develop-new-ui`.